gemfiles/ar_8_0.gemfile

```diff
@@ -0,0 +1,5 @@
+source "https://rubygems.org"
+gem 'activerecord', '~> 8.0.0'
+gem 'pg', '~> 1.1'
+
+gemspec path: "../"
```

Comment:
The `Appraisals` file should also be updated to include 7.2 and 8.0

Reply:
Oooh that's fancy. I see now how I could have solved my past Rails versioning issues